docs/release-notes/7.17.asciidoc

```diff
@@ -2,14 +2,24 @@
 == 7.17
 
 [discrete]
-[[release-notes-7.17.1]]
-=== 7.17.1
+[[release-notes-7.17.2]]
+=== 7.17.2
 
 [discrete]
-[[bug-fixes-7.17.1]]
+[[bug-fixes-7.17.2]]
 ==== Bug fixes and enhancements
+* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted under certain circumstances.
+* Ensures {endpoint-sec} can continue to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from `ELASTICSEARCH B.V.` to `Elasticsearch, Inc.` (https://github.com/elastic/endpoint/issues/15[#15]).
```

Comment:
```suggestion
* Ensures {endpoint-sec} continues to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from `ELASTICSEARCH B.V.` to `Elasticsearch, Inc.` (https://github.com/elastic/endpoint/issues/15[#15]).
```

Reply:
I'll apply this across the board. Thanks, @benironside !